packages/botonic-react/src/utils.js

```diff
@@ -82,19 +86,50 @@ export function msgToBotonic(msg) {
 }
 
 function quickreplies_parse(msg) {
-  let replies = msg.replies.map((el, i) => {
-    return (
-      <Reply key={i} payload={el.payload}>
-        {el.text}
-      </Reply>
-    )
-  })
+  let replies = null
+  let repliesKeyboard = null
+  if (msg.replies) {
+    msg.replies.map((el, i) => {
+      return (
+        <Reply key={i} payload={el.payload}>
+          {el.text}
+        </Reply>
+      )
+    })
+  }
+  if (msg.keyboard) {
+    msg.keyboard.map(el => {
+      return <Reply payload={el.data}>{el.label}</Reply>
```

Comment:
noob question: do we need a key attribute here?

Reply:
yes, it's a React thing: https://reactjs.org/docs/lists-and-keys.html